packages/web/src/utils.ts

```diff
@@ -0,0 +1,9 @@
+export function isBrowser(): boolean {
+  return typeof window !== 'undefined';
+}
+
+export function isProduction(): boolean {
+  return (
+    typeof process !== 'undefined' && process.env.NODE_ENV === 'production'
```

Comment:
Should we default to prod here? If process is not defined, I prefer to use the production version